docs/content/en/docs-dev/user-guide/managing-application/configuration-drift-detection.md

```diff
@@ -52,6 +52,8 @@ This status means the application is deploying and the configuration drift detec
 
 This feature is automatically enabled for all applications.
 
+If you want to trigger deployment automatically when `OUT_OF_SYNC` occurs, see [Trigger configuration](./triggering-a-deployment/#trigger-configuration).
```

Comment:
This should be a note and place under L57
```
This feature is automatically enabled for all applications.

You can change the checking interval as well as [configure the notification](../../managing-piped/configuring-notifications/) for these events in `piped` configuration.

Note: If you want to trigger deployment automatically when `OUT_OF_SYNC` occurs, see [Trigger configuration](./triggering-a-deployment/#trigger-configuration).
```

Reply:
thanks, i fixed [7b6dba3](https://github.com/pipe-cd/pipecd/pull/5120/commits/7b6dba380c85d6b8e0afa03b2c3806b521bcc67e)